Cargo.lock

```diff
@@ -44,7 +44,7 @@ version = "0.2.14"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d9b39be18770d11421cdb1b9947a45dd3f37e93092cbf377614828a319d5fee8"
 dependencies = [
- "hermit-abi",
+ "hermit-abi 0.1.19",
```

Comment:
Note: This'll likely fail in $INTERNAL because of cargo-deny.

Reply:
Do you know how this can be avoided?

Reply:
Hmm, not sure how I can revert that 🤔